bower.json

```diff
@@ -14,6 +14,7 @@
     "dist/angular-leaflet-directive.js"
   ],
   "dependencies": {
+    "Leaflet.vector-markers": "https://github.com/hiasinho/Leaflet.vector-markers",
```

Comment:
Oh, I did not try ´´vector-markers´´ solely. I thought packages are registered at Bower with the name that is denoted in the bower.json file.

Just to get it right:
- You prefer Bower packages over raw GitHub links?
- You prefer using Git tags over SHA# and SHA# over no version identifier at all?